pkg/resources/statefulsets/prometheus.go

```diff
@@ -59,7 +59,8 @@ func prometheusEnvVars(t *miniov1.Tenant) []corev1.EnvVar {
 func prometheusConfigVolumeMount(t *miniov1.Tenant) corev1.VolumeMount {
 	return corev1.VolumeMount{
 		Name:      t.PrometheusConfigVolMountName(),
-		MountPath: "/etc/prometheus",
+		MountPath: "/etc/prometheus/prometheus.yml",
+		SubPath:   "prometheus.yml",
```

Comment:
```suggestion
		SubPath:   "prometheus",
```

Right?

Reply:
This makes it so I only mount the `prometheus.yml` from the configmap into `/etc/prometheus/prometheus.yml`, if we switch to subpath `prometheus` the yml file will be mounted as a extensionless file

Reply:
Ok sounds good.